### ISSUE TITLE
File converted without audio

### ISSUE THREAD

**Issue by tjfriese** [mdhiggins/sickbeard_mp4_automator#1551]
File is converted without audio. I had updated to a version downloaded today and noticed new files seem to be created without an audio track. VLC shows no audio track on the file.

autoprocess.ini:
https://pastebin.com/Fv8FvBQ3

**Log files**
https://pastebin.com/VgM0UQyz

**Comment by mdhiggins** [comment]
Audio options are definitely included in the FFmpeg command

```
    "audio": [
        {
            "map": 1,
            "codec": "aac",
            "channels": 2,
            "bitrate": 256,
            "profile": null,
            "quality": 0,
            "filter": null,
            "samplerate": null,
            "sampleformat": "",
            "language": "eng",
            "disposition": "+default-dub-original-comment-lyrics-karaoke-forced-hearing_impaired-visual_impaired",
            "bsf": null,
            "title": "Stereo",
            "debug": "universal-audio"
        }
    ],
```

```
-c:a:0 aac -map 0:1 -ac:a:0 2 -b:a:0 256k -metadata:s:a:0 BPS=256000 -metadata:s:a:0 BPS-eng=256000 -metadata:s:a:0 title=Stereo -metadata:s:a:0 handler_name=Stereo -metadata:s:a:0 language=eng -disposition:a:0 +default-dub-original-comment-lyrics-karaoke-forced-hearing_impaired-visual_impaired-captions -strict experimental
```

I can't verify if there's audio or not, I would some info on the output file created (ffprobe -show_streams) but I'll take your word for it

I would also recommend you take the FFMPEG command given in your logs and run it manually to see if there's any errors or debug information that indicates its not making the audio for some reason

Looking at the other issue you opened, your FFmpeg build headers show

`--disable-decoder=eac3`

Meaning its probably failing to decode the eac3 audio track from your source file but just not completely cancelling the conversion like it otherwise would do when its missing the video decoder

Long story short, FFMPEG problem again, get/compile a different build that supports decoders for the input files you're feeding it
